tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -193,13 +227,39 @@ def testPluginIsActive(self):
     self._GenerateProjectorTestData()
     self._SetupWSGIApp()
 
-    # Embedding data is available.
+    # The is_active method makes use of a separate thread, so we mock threading
+    # behavior to make this test deterministic.
+    tf.test.mock.patch('threading.Thread', FakeThread).start()
```

Comment:
I think the one hurdle here is that the mock doesn't actually run the logic of the thread, so `self.assertTrue(self.plugin.is_active())` fails. Do you know of a way around that?